src/Components/toolbar/toolbar.js

```diff
@@ -0,0 +1,141 @@
+import 'filepond/dist/filepond.min.css';
+import { FontSizes, FontWeights, getTheme, mergeStyleSets, Modal } from 'office-ui-fabric-react';
+import { CommandBar } from 'office-ui-fabric-react/lib/CommandBar';
+import { initializeIcons } from 'office-ui-fabric-react/lib/Icons';
+import React, { Component } from 'react';
+import { FilePond } from 'react-filepond';
+import './toolbar.css';
+
+//icons for fabric ui
+initializeIcons();
+
+const theme = getTheme();
+const contentStyles = mergeStyleSets({
+    container: {
+      display: 'flex',
+      flexFlow: 'column nowrap',
+      alignItems: 'stretch'
+    },
+    header: [
+      theme.fonts.xLargePlus,
+      {
+        flex: '1 1 auto',
+        borderTop: `4px solid ${theme.palette.themePrimary}`,
+        color: theme.palette.neutralPrimary,
+        display: 'flex',
+        fontSize: FontSizes.xLarge,
+        alignItems: 'center',
+        fontWeight: FontWeights.semibold,
+        padding: '12px 12px 14px 24px'
+      }
+    ],
+    body: {
+      flex: '4 4 auto',
+      padding: '0 24px 24px 24px',
+      overflowY: 'hidden',
+      selectors: {
+        p: {
+          margin: '14px 0'
+        },
+        'p:first-child': {
+          marginTop: 0
+        },
+        'p:last-child': {
+          marginBottom: 0
+        }
+      }
+    }
+  });
+  
+
+export default class Toolbar extends Component {
+
+    state = {
+        isModalOpen: false,
+        fileUploaded: false,
+        files: [],
+        fileCount: 0
+    }
+
+    handleUpload = () => {
+        this.setState({isModalOpen: true})
+    }
+
+    _closeModal = () => {
```

Comment:
any specific reason for naming the functions starting with `_`, prefer normal camelCase approach as you did in other cases.